src/cfml/system/modules/coldbox-commands/templates/modules/ModuleConfig.cfc

```diff
@@ -60,6 +60,8 @@ Optional Methods
 	this.modelNamespace		= "@modelNamespace@";
 	// CF Mapping
 	this.cfmapping			= "@cfmapping@";
+	// Auto-map models
+	this.autoMapModels		= false;
```

Comment:
autoMapModels is easy but this also create lot of confusion once we hit the same name of CFC. 
App
--modules
---brand
----models
----Reference.cfc
----account
------Reference.cfc
----company
------Reference.cfc

think about this